nonsoolmateClient/src/home/homeTest/HomeTestSet.tsx

```diff
@@ -1,7 +1,129 @@
-import React from 'react'
+import styled from "styled-components";
+import { SetUnsetContainerLayout } from "style/layout/SetUnsetLayout";
+import { DownArrowBoldIc, SearchIc, UpArrowBoldIc } from "@assets/index";
+import { commonFlex } from "style/commonStyle";
+import { useState } from "react";
+import { universityLists } from "home/core/universityLists";
+import SelectedUniversityToggle from "./components/SelectedUniversityToggle";
 
 export default function HomeTestSet() {
+  const [selectedUniversityId, setSelectedUniversityId] = useState<number | null>(null);
+
+  function handleSelectedUniversityId(id: number) {
+    setSelectedUniversityId((prevId) => (prevId === id ? null : id));
+  }
+
   return (
-    <div>HomeTestSet</div>
-  )
+    <HomeTestSetContainer>
+      <HeaderBox>
+        <HeaderText>나의 시험장</HeaderText>
+        <HeaderButton type="button">
+          대학별 시험 찾기
+          <SearchIcon />
+        </HeaderButton>
+      </HeaderBox>
+      <SelectedUniversityLists>
+        {universityLists.map((data) => {
+          const { universityId, universityName, universityCategory, examList } = data;
+          const isSelected = selectedUniversityId === universityId;
+
+          return (
+            <div key={universityId}>
+              <SelectedUniversityButton
+                type="button"
+                onClick={() => handleSelectedUniversityId(universityId)}
+                $isSelected={isSelected}>
+                <UniversityBox>
+                  <Name $isSelected={isSelected}>{universityName}</Name>
+                  <Category $isSelected={isSelected}>{universityCategory}</Category>
+                </UniversityBox>
+                {isSelected ? <UpArrowBoldIcon /> : <DownArrowBoldIcon />}
+              </SelectedUniversityButton>
+              {isSelected && <SelectedUniversityToggle universityId={universityId} examList={examList} />}
+            </div>
```

Comment:
```suggestion
            <>
              <SelectedUniversityButton
                type="button"
                onClick={() => handleSelectedUniversityId(universityId)}
                $isSelected={isSelected}>
                <UniversityBox>
                  <Name $isSelected={isSelected}>{universityName}</Name>
                  <Category $isSelected={isSelected}>{universityCategory}</Category>
                </UniversityBox>
                {isSelected ? <UpArrowBoldIcon /> : <DownArrowBoldIcon />}
              </SelectedUniversityButton>
              {isSelected && <SelectedUniversityToggle universityId={universityId} examList={examList} />}
            </>
```
요기 이렇게 빈 태그로 바꿔줘도 될거 같은데 한 번 해볼래욤??